package-overrides/github/allpro/layout@1.4.3.json

```diff
@@ -9,7 +9,7 @@
     },
     "shim": {
         "jquery-ui-layout": {
-            "deps": "jquery"
+            "deps": ["jquery","jquery-ui"]
```

Comment:
I already added it last week or so : https://github.com/jspm/registry/blob/master/registry.json#L134